src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/WorkspaceProjectStateChangeDetector.cs

```diff
@@ -108,11 +108,10 @@ private void EnsureWorkQueue()
 
                     if (_workQueue is null)
                     {
-                        var errorReporter = _projectManager.Workspace.Services.GetRequiredService<ErrorReporter>();
```

Comment:
ErrorReporter is a new service and wasn't previously available in VSCode, resulting in this line throwing an exception.